toolflow/vivado/platform/zedboard/zedboard.tcl

```diff
@@ -20,6 +20,12 @@
 source -notrace $::env(TAPASCO_HOME_TCL)/platform/zynq/zynq.tcl
 
 namespace eval ::platform {
+
+  if { [::tapasco::vivado_is_newer "2021.1"] == 1 } {
+    puts "Vivado [version -short] is too new to support zedboard."
```

Comment:
Maybe we should find a more long-term approach to this? I assume that the issue is a missing board definition in Vivado. We could consider adding the board files to Tapasco (similar to ultra96) or dropping Zynq 7000 support completely.

Reply:
No, the issue does not seem to be missing board parts. The error occurs later, after creating a clocking wizard:
```
[10:30:24 <Thread-6: VivadoComposer> INFO] Creating component clk_wiz ...
[10:30:24 <Thread-6: VivadoComposer> INFO]   VLNV: xilinx.com:ip:clk_wiz:6.0
[10:30:28 <Thread-6: VivadoComposer> INFO] ERROR: [Common 17-167] Type error
```

Reply:
Nevermind, early there is also
```
WARNING: [Vivado 12-4842] No board parts matched 'get_board_parts -filter {NAME =~ "em.avnet.com:zed:part0*"}'.
Selected board part
```
So maybe it selects an incorrect board part and thus the error later

Reply:
As this is an unrelated issue, we should probably deal with it separately.